packtools/sps/validation/journal_meta.py

```diff
@@ -166,11 +166,47 @@ def validate_publisher_names(self, publisher_name_list):
         if not publisher_name_list:
             raise ValidationPublisherException("Function requires a list of publisher names")
 
+        expected = publisher_name_list
+        obtained = self.publisher.publishers_names
 
+        for publisher in zip(expected, obtained):
+            is_valid = publisher[0] == publisher[1]
             yield {
                     'title': 'Publisher name element validation',
                     'xpath': './/publisher//publisher-name',
                     'validation_type': 'value',
+                    'response': 'OK' if is_valid else 'ERROR',
+                    'expected_value': publisher[0],
+                    'got_value': publisher[1],
+                    'message': 'Got {} expected {}'.format(publisher[1], publisher[0]),
+                    'advice': None if is_valid else 'The publisher name in the XML does not match what was expected, '
```

Comment:
`"Provide the expected publisher name: {expected}"`

Reply:
No advice, apenas dê a orientação, a justificativa já está em message.

Reply:
OK, @robertatakenaka